src/MainWindow.vala

```diff
@@ -20,6 +20,10 @@ public class MainWindow : Gtk.ApplicationWindow {
     }
 
     construct {
+        var provider = new Gtk.CssProvider ();
```

Comment:
You actually don't need to do this step at all anymore. Granite automatically loads gresourced stylesheets called "Application.css"